arboretum/permissions/README.md

```diff
@@ -94,6 +94,55 @@ how to include the fetchers and checks from this library in your downstream proj
 
 ## Checks
 
+### Organization Integrity (Repository Permissions)
+
+* Class: [GithubOrgPermissionsCheck][org-permissions-check]
+* Purpose: Control the access to Github repositories containing source code.
+* Behavior: Collaborators, fork and teams are checked for every repository.
+Each direct collaborator found in a repo will be considered a failure. Each fork found in a repo will be considered a warning. As additional information the permissions check also lists the organization teams as successes.
```

Comment:
We should not be listing successes in check results or reports.  This is how you wind up with check results that are too big to be rendered in GH.  The purpose of a check is to flag warnings and failures so that they can be remediated not to list everything that right in the world.

Reply:
Done